bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Semester.scala

```diff
@@ -65,8 +65,8 @@ case class Semester(year: Int, half: SemesterOption) {
     cal.setTimeZone(TimeZone.getTimeZone("UTC"))
     cal.set(Calendar.YEAR, year)
     cal.set(Calendar.MONTH, half.firstMonth)
-    cal.set(Calendar.DAY_OF_MONTH, 0)
-    cal.set(Calendar.HOUR, 0)
+    cal.set(Calendar.DAY_OF_MONTH, 1)
+    cal.set(Calendar.HOUR_OF_DAY, 0)
```

Comment:
In the core `Semester` implementation the start date is defined as mentioned above and the end date is calculated by adding 6 months to the start date via `Calendar.add`.  Would it be possible to just use the core `Semester` instead of having a different version?  I think the P1 version was split off back when the PIT was a separate codebase but presumably there's no need to keep different `Semester` implementations now?